packages/wonder-blocks-modal/util/maybe-get-portal-mounted-modal-host-element.js

```diff
@@ -29,11 +29,5 @@ function maybeGetNextAncestorModalLauncherPortal(element: ?(Element | Text)) {
 export default function maybeGetPortalMountedModalHostElement(
```

Comment:
Looks like we can just get rid of this function - it's not doing anything now!

Reply:
I think maybe this can go away once everything is using new portals and we've confirmed the modal isn't weirdly clipping other portal'd things.

Reply:
I think we can get rid of this eventually but it's a non-trivial amount of work b/c we need to figure out a different way of figuring out what a portal's host/container element should be.

Reply:
I'll add a TODO.